extensions/notebook/src/extension.ts

```diff
@@ -35,7 +35,7 @@ export async function activate(extensionContext: vscode.ExtensionContext): Promi
 	 * If changes are made to bookTreeView.openBook, please ensure backwards compatibility with its current state.
 	 * This is the command used in the extension generator to open a Jupyter Book.
 	 */
-	extensionContext.subscriptions.push(vscode.commands.registerCommand('bookTreeView.openBook', (bookPath: string, openAsUntitled: boolean, urlToOpen?: string) => openAsUntitled ? providedBookTreeViewProvider.openBook(bookPath, urlToOpen, true) : bookTreeViewProvider.openBook(bookPath, urlToOpen, true)));
+	extensionContext.subscriptions.push(vscode.commands.registerCommand('bookTreeView.openBook', (bookPath: string, openAsUntitled: boolean, urlToOpen?: string, isNotebook?: boolean) => openAsUntitled ? providedBookTreeViewProvider.openBook(bookPath, urlToOpen, true, isNotebook) : bookTreeViewProvider.openBook(bookPath, urlToOpen, true, isNotebook)));
```

Comment:
openBook can also open Notebooks now? Same as all the others it would be preferable if we could have the command here actually fit what it's doing